app/templates/ember_charts/time_series.hbs

```diff
@@ -92,9 +94,25 @@
             </div>
           </div>
         </div>
+        <div class="form-group">
+          <div class="col-lg-offset-4 col-lg-8">
+            <div class="checkbox">
+              <label>{{view Ember.Checkbox checkedBinding="dynamicXAxis"}} Use Dynamic X-Axis</label>
```

Comment:
I think we may have problem here with the appearance. The default checkbox might look different than our internal one. Can you take screenshots of how it looks like in the demo website (not our app) and share it in the PR description?